app/jobs/migration/conversations_first_reply_scheduler_job.rb

```diff
@@ -0,0 +1,17 @@
+# Delete migration and spec after 2 consecutive releases.
+class Migration::ConversationsFirstReplySchedulerJob < ApplicationJob
+  queue_as :scheduled_jobs
+
+  def perform(account)
+    account.conversations.each do |conversation|
+      # rubocop:disable Rails/SkipsModelValidations
+      if conversation.messages.outgoing.where("(additional_attributes->'campaign_id') is null").count.positive?
+        conversation.update_columns(first_reply_created_at: conversation.messages.outgoing.where("(additional_attributes->'campaign_id') is null")
+        .first.created_at)
```

Comment:
Can we have this in the method? it would be more readable that way.

Reply:
We decided to delete this migration after 2 consecutive releases. So @tejaswinichile do we need to do this?

Reply:
> We decided to delete this migration after 2 consecutive releases. So @tejaswinichile do we need to do this?

Then it's fine :D